lib/std/math/pow.zig

```diff
@@ -35,8 +35,8 @@ pub fn pow(comptime T: type, x: T, y: T) T {
         return math.powi(T, x, y) catch unreachable;
     }
 
-    if (T != f32 and T != f64) {
-        @compileError("pow not implemented for " ++ @typeName(T));
+    if (@typeInfo(T) == .comptime_float) {
+        @compileError("pow not implemented for comptime_float");
```

Comment:
```suggestion
    if (@typeInfo(T) != .float) {
        @compileError("pow not implemented for comptime_float");
```

per your current semantics I think this would be better. you could also leave the check out entirely and leave `std.math.floatFractionalBits` to surface the compile error which would have the same effect